frappe/patches/v12_0/move_email_and_phone_to_child_table.py

```diff
@@ -52,7 +52,7 @@ def execute():
 			phone_values.append((
 				phone_counter,
 				frappe.generate_hash(contact_detail.email_id, 10),
-				contact_detail.phone,
```

Comment:
The patch was meant to move both phone and mobile_no to the child table

Reply:
@hrwX at line 38 we are already moving phone number and in the block which is meant for mobile number we are again moving phone number making it entered twice. When we ran migration all our phone numbers were entered twice in the contact table.